samples/qt/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.19)
+project(qt LANGUAGES CXX)
+
+find_package(Qt6 6.5 REQUIRED COMPONENTS Core)
+
+qt_standard_project_setup()
+
+qt_add_executable(qt
+    main.cpp
+)
+
+target_link_libraries(qt
+    PRIVATE
+        Qt::Core
+)
+
+include(GNUInstallDirs)
+
+install(TARGETS qt
+    BUNDLE  DESTINATION .
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+
+qt_generate_deploy_app_script(
+    TARGET qt
+    OUTPUT_SCRIPT deploy_script
+    NO_UNSUPPORTED_PLATFORM_ERROR
+)
+install(SCRIPT ${deploy_script})
+
+include(FetchContent)
+FetchContent_Declare(
+    astarte_device_sdk
+  GIT_REPOSITORY git@github.com:secomind/astarte-device-sdk-cpp.git
+  GIT_TAG main
+)
+FetchContent_MakeAvailable(astarte_device_sdk)
```

Comment:
In our samples, we generally do not import the SDK, though fetch content. We instead add the root SDK directory as a subdirectory.
We want to prevent people building the sample from one branch and thinking it would execute with its local code while it actually builds with `main`.